clarity/fuzz/Cargo.toml

```diff
@@ -0,0 +1,51 @@
+[package]
+name = "clarity-fuzz"
+version = "0.0.0"
+publish = false
+edition = "2021"
+
+[package.metadata]
+cargo-fuzz = true
+
+[dependencies]
+libfuzzer-sys = "0.4"
+# clarity_2_1 = { git = "https://github.com/stacks-network/stacks-blockchain.git", tag = "2.1.0.0.0", package="clarity"}
+
+[dependencies.clarity]
+path = ".."
+features = ["testing"]
+
+[dependencies.stacks-common]
+path = "../../stacks-common"
+features = ["testing"]
+
+# Prevent this from interfering with workspaces
+[workspace]
+members = ["."]
+
+[profile.release]
+debug = 1
+
+[[bin]]
+name = "fuzz_sanitize"
+path = "fuzz_targets/fuzz_sanitize.rs"
+test = false
+doc = false
+
+# [[bin]]
```

Comment:
nit: Are these just TODOs, or are the artifacts of a change that is no longer meant to be included? Either way, commented-out code is confusing.

Reply:
Removed -- these were artifacts from the prior fuzzing draft PRs.